dll/cpl/joy/joy.c

```diff
@@ -23,12 +23,12 @@
  * PURPOSE:         ReactOS Software Control Panel
  * PROGRAMMER:      Dmitry Chapyshev (lentind@yandex.ru)
  * UPDATE HISTORY:
```

Comment:
Add your name here

Reply:
Content yes, name no (when i'll be a dev here :-) )